framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -374,8 +437,47 @@ public static boolean canInsertJavaExpression(JavaExpression expr) {
      * new and old value are taken (according to the lattice). Note that this happens per hierarchy,
      * and if the store already contains information about a hierarchy for which {@code value} does
      * not contain information, then that information is preserved.
+     *
+     * <p>This method does nothing if {@code expr} cannot be inserted into the store.
+     *
+     * @param expr the expression to insert in the store
+     * @param value the value of the expression
+     */
+    public final void insertValue(JavaExpression expr, @Nullable V value) {
```

Comment:
You have changed the semantics of this method.  That should be mention in the change log.